public-ui/src/app/data-browser/concept-charts/concept-charts.component.html

```diff
@@ -1,9 +1,27 @@
-<div class="db-page-spinner" *ngIf="loading()">
+<div class="db-page-spinner" *ngIf="loading() && !displayGraphErrorMessage">
   <span class="spinner">
     Loading...
   </span>
 </div>
 
+<div *ngIf="displayGraphErrorMessage">
```

Comment:
This outer div seems unnecessary here. Can we remove it and put the `*ngIf` on the next one down (graph-error-message)?

Reply:
Removed this. Thanks.